.docs/README.md

```diff
@@ -276,25 +257,20 @@ try {
 }
 ```
 
-### Po platbě
+### After payment
 
-Váš zákazník provede potřebné úkony na Gopay platební bráně, a jakmile je proces
-dokončen, je přesměrován zpátky do vaší aplikace, buď na `successUrl`
-nebo `failureUrl` adresu. Obě dvě dostanou od Gopay následující sadu parametrů:
+Your customer will perform the necessary actions on the Gopay payment gateway, and once the process is done completed, is redirected back to your application, either to `successUrl` or `failureUrl` address. Both will receive the following set of parameters from Gopay:
```

Comment:
Your customer will perform the necessary actions on the Gopay payment gateway, and once the process is completed,  he is redirected back to your application, either to `successUrl` or `failureUrl` address. Both will receive the following set of parameters from Gopay: